components/product/ProductCard.tsx

```diff
@@ -94,26 +114,103 @@ function ProductCard({ product, preload, itemListName, layout }: Props) {
       </a>
     </li>
   ));
-  const cta = (
-    <a
-      href={url && relative(url)}
-      aria-label="view product"
-      class="btn btn-block"
-    >
-      {l?.basics?.ctaText || "Ver produto"}
-    </a>
-  );
+
+  const cta = layout?.basics?.ctaMode === "Go to Product Page"
+    ? (
+      <a
+        href={url && relative(url)}
+        aria-label="view product"
+        style={{
+          minWidth: "162px",
+        }}
+        class={`lg:text-sm font-medium text-xs whitespace-nowrap m-auto btn max-md:min-h-[2.25rem] max-md:h-[2.25rem] btn-${
```

Comment:
Voce pode evitar a tag style utilizando os valores dinamicos do tailwind
```suggestion
        class={`lg:text-sm min-w-[162px] font-medium text-xs whitespace-nowrap m-auto btn max-md:min-h-[2.25rem] max-md:h-[2.25rem] btn-${
```